packages/terra-dialog-modal/examples/Index.site-page.jsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions */
+import React from 'react';
+import DocTemplate from 'terra-doc-template';
+
+import ReadMe from '../docs/README.md';
+import { name } from '../package.json';
+
+// Component Source
+// eslint-disable-next-line import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions
+import DialogModalSrc from '!raw-loader!../src/DialogModal';
```

Comment:
The file structure/naming as been updated in #175 and with this, we altered how we pull in examples to enable hot reloading during development. 
```diff
component
+ move and rename to these too:
   /examples                    -> src/terra-dev-site/
        /test-examples          -> src/terra-dev-site/test/dialog-modal/.
            /DefaultExample     -> src/terra-dev-site/test/dialog-modal/DefaultExample.test.jsx
        /index-examples         -> src/terra-dev-site/doc/examples/.
       Index.site-page.jsx      -> src/terra-dev-site/doc/Dialog.doc.jsx    
```


Here's an example of 
doc (Index.site-page.jsx) example imports: https://github.com/cerner/terra-framework/blob/master/packages/terra-hookshot/src/terra-dev-site/doc/Hookshot.doc.jsx#L8

Here is an example of test imports: https://github.com/cerner/terra-framework/blob/master/packages/terra-hookshot/src/terra-dev-site/test/hookshot/DefaultHookshot.test.jsx#L2

Reply:
Changed this while consuming terra-dev-site 1.0